tools/release_builder.py

```diff
@@ -24,19 +24,19 @@
 class UniverseReleaseBuilder(object):
 
     def __init__(self, package_version, stub_universe_url,
-                 commit_desc = '',
-                 min_dcos_release_version = os.environ.get('MIN_DCOS_RELEASE_VERSION', '1.8'),
-                 http_release_server = os.environ.get('HTTP_RELEASE_SERVER', 'https://downloads.mesosphere.com'),
-                 s3_release_bucket = os.environ.get('S3_RELEASE_BUCKET', 'downloads.mesosphere.io'),
-                 release_docker_image = os.environ.get('RELEASE_DOCKER_IMAGE'),
-                 release_dir_path = os.environ.get('RELEASE_DIR_PATH', ''),
-                 beta_release = os.environ.get('BETA', 'False')):
+                 commit_desc='',
+                 min_dcos_release_version=os.environ.get('MIN_DCOS_RELEASE_VERSION', '1.8'),
+                 http_release_server=os.environ.get('HTTP_RELEASE_SERVER', 'https://downloads.mesosphere.com'),
+                 s3_release_bucket=os.environ.get('S3_RELEASE_BUCKET', 'downloads.mesosphere.io'),
+                 release_docker_image=os.environ.get('RELEASE_DOCKER_IMAGE'),
+                 release_dir_path=os.environ.get('RELEASE_DIR_PATH', ''),
+                 beta_release=os.environ.get('BETA', 'False')):
         self._dry_run = os.environ.get('DRY_RUN', '')
-        self._force_upload = bool(os.environ.get('FORCE_ARTIFACT_UPLOAD', 'false'))
-        name_match = re.match('.+/stub-universe-(.+).zip$', stub_universe_url)
+        self._force_upload = bool(os.environ.get('FORCE_ARTIFACT_UPLOAD', ''))
```

Comment:
I really don't like this pattern of using empty string as false.  
None, False are good.

Here though, 
`self._force_upload = 'FORCE_ARTIFACT_UPLOAD' in os.environ`
or if you prefer
`self._force_upload = os.environ.has_key('FORCE_ARTIFACT_UPLOAD')`